src/replication_handler/include/replication_handler/replication_handler.hpp

```diff
@@ -66,7 +64,7 @@ void SystemRestore(replication::ReplicationClient &client, system::System &syste
         return std::nullopt;
       });
 
-      if (full_system_replication) {
+      if (is_enterprise) {
```

Comment:
Did we decide which system replication is enterprise? 
Are both the multi-tenant and auth enterprise?
MT has to be, but AUTH could be community.

Reply:
Did behavior change with the introduced line changes?

Reply:
As I understand, purpose of this work is just to remove system replication out of experimental features and leave everything else as it was.

Reply:
Both are enterprise as I know.